### PR TITLE
Preparation for SnapshotTesting support

### PR DESCRIPTION
**Pull request by mackoj**
## Changes description

I added some tests to prevent inadvertent regressions, which will provide additional code coverage data.

Capture d’écran 2023-05-03 à 17 08 22

This force me to raise the Showcase minimum version to iOS 13.

## Context

I'm trying to have more control over the codebase. To ensure safer acceptance of PRs, I prefer having a minimum amount of tests.

## Checklist
- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on an iPhone device/simulator.
- [x] I have tested on an iPad device/simulator.
- [x] If it includes design changes, please ask for a review `design-system-core-team-design` GitHub team.

### CODE REVIEW

File: Showcase/ShowcaseSnapshotTests/ShowcaseSnapshotSwiftUITests.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import XCTest
```
Comment (florentlotthepro) [inline Showcase/ShowcaseSnapshotTests/ShowcaseSnapshotSwiftUITests.swift:1]: On files in Sources, we wanted to have a header with the name of the project and the license. I don't know if we also want them here 🤔

File: Showcase/ShowcaseSnapshotTests/ShowcaseSnapshotUIKitTests.swift [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import XCTest
```
Comment (florentlotthepro) [inline Showcase/ShowcaseSnapshotTests/ShowcaseSnapshotUIKitTests.swift:1]: Same here

**Review by florentlotthepro — APPROVED** [review]
Nice work! 
Now, we can see some issues in components 😅

File: Showcase/ShowcaseSnapshotTests/ShowcaseSnapshotSwiftUITests.swift (hunk 1 above)
Comment (mackoj) [reply Showcase/ShowcaseSnapshotTests/ShowcaseSnapshotSwiftUITests.swift:1]: Ok I will fix it thanks for the review.

File: Showcase/ShowcaseSnapshotTests/ShowcaseSnapshotUIKitTests.swift (hunk 2 above)
Comment (mackoj) [reply Showcase/ShowcaseSnapshotTests/ShowcaseSnapshotUIKitTests.swift:6]: Ok I will fix it thanks for the review.

**Review by florentlotthepro — APPROVED** [review]

**Pull request merged**